src/main/java/seedu/address/model/person/Person.java

```diff
@@ -77,12 +77,23 @@ public boolean isSamePerson(Person otherPerson) {
     /**
      * Sets the person's archive status to true. It's equivalent to archive the person.
      *
-     * @return A Person type with "true" archive status.
+     * @return A Person type with "true" in archive status.
```

Comment:
Could be better phrased as: "A Person whose archive status is true"

Reply:
Yes! that will be better, thanks!